webpack.config.js

```diff
@@ -7,7 +7,12 @@ module.exports = {
     rules: [
       {
         test: /\.ts$/,
-        use: "ts-loader",
+        use: {
+          loader: "ts-loader",
+          options: {
+            transpileOnly: true,
+          },
+        },
```

Comment:
skips typescript checks as part of the webpack build.
build with the command;
`yarn webpack --config webpack.config.js`